src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/MeetTheSpecialists.tsx

```diff
@@ -0,0 +1,182 @@
+import {
+  Box,
+  Button,
+  Column,
+  Flex,
+  GridColumns,
+  Pill,
+  ResponsiveBox,
+  Text,
+  Image,
+} from "@artsy/palette"
+import {
+  SPECIALISTS,
+  Specialty,
+} from "Apps/Consign/Routes/MarketingLanding/Components/LandingPage/SpecialistsData"
+import { Rail } from "Components/Rail"
+import { useState } from "react"
+import { useTracking } from "react-tracking"
+import { RouterLink } from "System/Router/RouterLink"
+import { useAnalyticsContext, useSystemContext } from "System"
+
+interface PillData {
+  type: Specialty
+  title: string
+}
+
+const pills: PillData[] = [
+  {
+    type: "auctions",
+    title: "Auctions",
+  },
+  {
+    type: "priveteSalesAndAdvisory",
+    title: "Private Sales & Advisory",
+  },
+  {
+    type: "collectorServices",
+    title: "Collector Services",
+  },
+]
+
+export const MeetTheSpecialists: React.FC = () => {
+  const { user } = useSystemContext()
+  const { contextPageOwnerType } = useAnalyticsContext()
+  const { trackEvent } = useTracking()
+
+  const [selectedSpecialty, setSelectedSpecialty] = useState<Specialty>(
+    "auctions"
+  )
+  const [specialistsTooDisplay, setSpecialistsTooDisplay] = useState(
+    SPECIALISTS.filter(i => i.specialty === "auctions")
+  )
+
+  const contactTheSpecialistClick = () => {
+    trackEvent({
+      action: "clickedGetInTouch",
+      context_module: "MeetTheSpecialists",
+      context_page_owner_type: contextPageOwnerType,
+      label: "Contact specialist_name",
+      user_id: user?.id,
+      user_email: user?.email,
+    })
+  }
+
+  const trackStartSellingClick = () => {
+    trackEvent({
+      action: "clickedStartSelling",
+      context_module: "MeetTheSpecialists",
+      context_page_owner_type: contextPageOwnerType,
+      label: "Start Selling",
+      destination_path: "/sell/submission",
+      user_id: user?.id,
+    })
+  }
+
+  return (
+    <>
+      <Text mb={[0.5, 1]} variant={["lg-display", "xxl"]}>
+        Meet the specialists
+      </Text>
+      <Text mb={[2, 4]} variant={["xs", "sm"]}>
+        Our in-house experts cover Post-War and Contemporary Art, Prints and
+        Multiples, Street Art and Photographs.
+      </Text>
+      <Flex overflowY="scroll">
+        {pills.map(pill => (
+          <Pill
+            mr={[1, 2]}
+            hover={false}
+            selected={selectedSpecialty === pill.type}
+            variant="default"
+            onClick={() => {
+              setSelectedSpecialty(pill.type)
+              setSpecialistsTooDisplay(
+                SPECIALISTS.filter(i => i.specialty === pill.type)
+              )
+            }}
+          >
+            {pill.title}
+          </Pill>
+        ))}
+      </Flex>
+      <Rail
+        title=""
```

Comment:
**question:** Is this empty title needed?

Reply:
I think so, it's a required prop for the Rail component